<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>557</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>221</width>
      <height>316</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="processVerticalLayout">
     <item>
      <widget class="QPushButton" name="selectFolderPushButton">
       <property name="text">
        <string>Selectionner dossier</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="typedeFichierLabel">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Type de fichier</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QCheckBox" name="optionsImageCheckBox">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Image</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QCheckBox" name="optionsVideoCheckBox">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Video</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="optionsDocumentCheckBox">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Document</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QCheckBox" name="optionsAudioCheckBox">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Audio</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="optionsArchiveCheckBox">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Archive</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="FiltreLabel">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Filtre</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="filterNameCheckBox">
       <property name="text">
        <string>Comparer nom de fichier</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="filterSizeAndDateCheckBox">
       <property name="text">
        <string>Comparer taille et date</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="filterSHA256MD5CheckBox">
       <property name="text">
        <string>SHA-256</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QCheckBox" name="filterPHashCheckBox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>pHash</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="pHashThresholdSpinBox">
         <property name="maximum">
          <number>15</number>
         </property>
         <property name="value">
          <number>5</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="deleteDoublonsPushButton">
       <property name="text">
        <string>Supprimer doublons</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QListWidget" name="logUserListWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>311</y>
      <width>221</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="QProgressBar" name="filterProgressBar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>527</y>
      <width>221</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
